ArjunaJTA/jta/classes/com/arjuna/ats/jta/common/JTAEnvironmentBean.java

```diff
@@ -586,30 +585,6 @@ public void setXaResourceOrphanFilters(List<XAResourceOrphanFilter> xaResourceOr
         }
     }
 
-    /**
-     * Returns if connections associated to XAResources that fail during prepare should be cleaned up immediately.
```

Comment:
@jmfinelli I think the Javadoc of the setter and getter should be updated as the behaviour described is no longer expected to take any effect at runtime.

It should also be `@deprecated` in the Javadoc and @Deprecated in the code

Reply:
Done

Reply:
Thank you!